src/content/supported-icons.tsx

```diff
@@ -35,14 +35,19 @@ import { IconTerminal16 } from '@hashicorp/flight-icons/svg-react/terminal-16'
 import { IconTerminalScreen16 } from '@hashicorp/flight-icons/svg-react/terminal-screen-16'
 import { IconTerraformColor16 } from '@hashicorp/flight-icons/svg-react/terraform-color-16'
 import { IconTools16 } from '@hashicorp/flight-icons/svg-react/tools-16'
-import { IconVaultColor16 } from '@hashicorp/flight-icons/svg-react/vault-color-16'
+import { IconVault16 } from '@hashicorp/flight-icons/svg-react/vault-16'
 import { IconVmware16 } from '@hashicorp/flight-icons/svg-react/vmware-16'
 import { IconWaypointColor16 } from '@hashicorp/flight-icons/svg-react/waypoint-color-16'
 import { IconWrench16 } from '@hashicorp/flight-icons/svg-react/wrench-16'
 
+/**
+ * @TODO refactor to remove the HashiCorp product logos from this list
+ * when product logos are needed, component should use `ProductIcon`
+ */
+
 export const SUPPORTED_ICONS = {
 	api: <IconApi16 />,
-	'aws-color': <IconAwsColor16 />,
+	'aws-color': <IconAws16 />,
```

Comment:
```suggestion
	'aws': <IconAws16 />,
```
This shouldn't be color. This is what we should connect on.

Reply:
Write a note for context on aws being the outlier here